doc/source/contributor/environments/ci-aio.rst

```diff
@@ -48,10 +43,21 @@ Run the setup script:
    ./automated-setup.sh
 
 The script will pull the current version of Kayobe and this repository, and
-then run the manual setup steps below. The script can be easily edited to use
-a different branch of Kayobe or this repository. The script will automatically
-determine whether your image is LVM based, if so, it will expand the volume sizes
-to allow ansible dependencies to install correctly.
+then run the manual setup steps below. The script can be easily edited with the
+following options:
+
+* ``BASE_PATH`` (default: ``~``) - Directory to deploy from. The directory must
+  exist before running the script.
+* ``KAYOBE_BRANCH`` (default: ``stackhpc/2023.1``) - The branch of the Kayobe
+  source code to use. Must exist in the ``stackhpc/kayobe`` GitHub repository.
```

Comment:
> Must exist in the ``stackhpc/kayobe`` GitHub repository.

Does this need to be said? If so, why not say it for the kayobe-config too?

Reply:
From discussion in person:
I just wanted to make it clear that the kayobe branch is always pulled from our downstream repo, not upstream. I didn't say the same for kayobe-config because it's **stackhpc**-kayobe-config so it's easy to infer the same thing from the name. I do agree that it does look a bit weird as is so I'll rewrite it.

Reply:
Removed